src/Inviqa/Worldpay/Api/Request/ApplePayPaymentService/MerchantCode.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Inviqa\Worldpay\Api\Request\ApplePayPaymentService;
+
+use Inviqa\Worldpay\Api\XmlAttributeDefaults;
+
+class MerchantCode extends XmlAttributeDefaults
```

Comment:
A large number of value objects are the same across both standard Worldpay and ApplePay requests.
Is there any way of relying on the existing objects and only defining new ones for Apple, such as the Apple SSL ones?